custom_dc/env.list.sample

```diff
@@ -48,7 +49,8 @@ GOOGLE_ANALYTICS_TAG_ID=
 
 ### Variables that rarely need to be changed ###
 
-# Enable embeddings generation and natural language querying.
+# Enable embeddings generation and natural language querying. If you find that container startup 
```

Comment:
Let's not add the additional comment. They lose NL if they set it to false.

Reply:
SG.